google-beta/resource_pubsub_subscription_test.go

```diff
@@ -1,38 +1,9 @@
 package google
 
 import (
-	"fmt"
 	"testing"
-
-	"github.com/hashicorp/terraform/helper/acctest"
-	"github.com/hashicorp/terraform/helper/resource"
-	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestAccPubsubSubscription_basic(t *testing.T) {
-	t.Parallel()
-
-	topic := fmt.Sprintf("tf-test-topic-%s", acctest.RandString(10))
-	subscription := fmt.Sprintf("tf-test-sub-%s", acctest.RandString(10))
```

Comment:
Do you mind turning this case (or copying the example test in, either is fine) into `_fullNameTest` and providing a full name here?

`subscription := fmt.Sprintf("projects/getTestProjectFromEnv()/subscriptions/tf-test-sub-%s", acctest.RandString(10))`

or similar, and same with the topic (they can both be in the same test if you'd like)

The behaviour is pretty fiddly, so I want to make sure we get it right.